pulsar/internal/connection.go

```diff
@@ -36,6 +36,14 @@ import (
 	"github.com/apache/pulsar-client-go/pulsar/internal/pb"
 )
 
+const (
+	// TODO: Find a better way to embed the version in the library code
```

Comment:
Maybe we can create a `version` file and read the version value from the file

Reply:
Problem is that we're shipping this as a library, so I guess the version file is not going to be bundled. Anyway, as a start we could go by with remembering to update the version string in source code, until we find a better solution.